### PR TITLE
[1.29] feat: Deprecate 'syspurpose addons'

### PR DESCRIPTION
**Pull request by m-horky**
* Card ID: CCT-599

### CODE REVIEW

File: man/subscription-manager.8 [hunk 1]
```diff
@@ -396,17 +396,16 @@ The
 command has subcommands for all the various syspurpose preferences and attributes:
 
 .IP
-1. addons
+1. role
```
Comment (jirihnidek) [inline man/subscription-manager.8:399]: The command is deprecated in this version. Not removed.

**Review by jirihnidek — CHANGES_REQUESTED** [review]
Overall it looks good. Please rebase your feature branch with target branch. I have also small request related to the man page.

File: man/subscription-manager.8 (hunk 1 above)
Comment (m-horky) [reply man/subscription-manager.8:399]: That matches what we have done in the past for top-level commands. Deprecated ones mentioned below the main list, on one line.

https://github.com/candlepin/subscription-manager/blob/c2551d4df1cf4b38dbcc93848d052021020423f0/man/subscription-manager.8#L134-L141

Comment (jirihnidek) [reply man/subscription-manager.8:399]: Yeah, but this is list of syspurpose sub-commands and all sub-commands will be still the same in 1.29 branch.

**Comment by jirihnidek** [comment]
@m-horky is it still draft or is it ready for review again?

**Comment by m-horky** [comment]
@jirihnidek You can review it.

**Review by jirihnidek — APPROVED** [review]
LGTM

**Pull request merged**